src/views/tickets/orders.php

```diff
@@ -44,7 +44,13 @@
 	<form method="post">
 
 	<?php tribe_tickets_get_template_part( 'tickets/orders-rsvp' ); ?>
-	<?php tribe_tickets_get_template_part( 'tickets/orders-tickets' ); ?>
+
+		<?php
+		/**
+		 * Adds ticket order form
+		 */
+		do_action( 'event_tickets_plus_orders_tickets' );
```

Comment:
Nice - this seems like the right kind of approach but:

* (Minor) this block is indented too far, we should tab back one
* Naming is slightly off: within Event Tickets, the prefix we generally use for hooks is `tribe_tickets_*` so something like `tribe_tickets_orders_before_submit` might be preferable here (updating the comment to something more generic, such as _"Fires before the process tickets submission button is rendered")_